tests/parametric/test_otel_span_methods.py

```diff
@@ -1026,8 +1029,11 @@ def test_otel_record_exception_attributes_serialization(self, test_agent, test_l
         event3 = events[2]
         assert event3["attributes"].get("exception.message") == "message override"
 
-        error_message = root_span["meta"].get("error.message") or root_span["meta"].get("error.msg")
-        assert error_message == "message override"
+        # For PHP we set only the error.stack tag on the meta to not interfere with the defined semantics of the PHP tracer
+        # https://github.com/DataDog/dd-trace-php/pull/2754#discussion_r1704232289
+        if context.library != "php":
+            error_message = root_span["meta"].get("error.message") or root_span["meta"].get("error.msg")
+            assert error_message == "message override"
```

Comment:
Can you add a separate test that tests that all of the error tracking tags are set and then skip the entire test for PHP? Otherwise, I feel like we're going to forget that PHP has this differing behavior. Feel free to keep these test cases as-is though

Reply:
Added [here](https://github.com/DataDog/system-tests/pull/2841/commits/55f208511ec788ecc444c568d3c7b45de9364b92) let me know what you think, it also Node doesn't override based on attributes since they don't support that so I made it simple.